src/app/upgrade.cpp

```diff
@@ -75,10 +85,66 @@ namespace
             , QLatin1String("Preferences/WebUI/HTTPS/KeyPath")
             , Utils::Fs::toNativePath(configPath + QLatin1String("WebUIPrivateKey.pem")));
     }
+
+#if defined Q_OS_WIN
+    void upgradeResumeFiles()
```

Comment:
please add an link or short description about why this is necessary.

Reply:
Added description comment.